tests/model/model_flattening.cpp

```diff
@@ -591,393 +591,125 @@ TEST(ModelFlattening, importedComponentWithNameClashes)
-        "            <ci>i_K</ci>\n"
-        "            <ci>i_Na</ci>\n"
-        "            <ci>i_L</ci>\n"
-        "          </apply>\n"
-        "          <ci>Cm</ci>\n"
-        "        </apply>\n"
-        "      </apply>\n"
-        "    </math>\n"
-        "  </component>\n"
-        "  <component name=\"sodium_channel_1\">\n"
-        "    <variable name=\"V\" units=\"mV\" interface=\"public_and_private\"/>\n"
-        "    <variable name=\"t\" units=\"ms\" interface=\"public_and_private\"/>\n"
-        "    <variable name=\"h\" units=\"dimensionless\" interface=\"private\"/>\n"
-        "    <variable name=\"m\" units=\"dimensionless\" interface=\"private\"/>\n"
-        "    <variable name=\"g_Na\" units=\"mS_per_cm2\" initial_value=\"120\"/>\n"
-        "    <variable name=\"E_Na\" units=\"mV\"/>\n"
-        "    <variable name=\"i_Na\" units=\"microA_per_cm2\"/>\n"
-        "    <variable name=\"Nao\" units=\"mM\" initial_value=\"140\"/>\n"
-        "    <variable name=\"Nai\" units=\"mM\" initial_value=\"30\"/>\n"
-        "    <variable name=\"RTF\" units=\"mV\" initial_value=\"25\"/>\n"
-        "    <variable name=\"Na_conductance\" units=\"mS_per_cm2\"/>\n"
-        "    <math xmlns=\"http://www.w3.org/1998/Math/MathML\" xmlns:cellml=\"http://www.cellml.org/cellml/2.0#\">\n"
-        "      <apply>\n"
-        "        <eq/>\n"
-        "        <ci>E_Na</ci>\n"
-        "        <apply>\n"
-        "          <times/>\n"
-        "          <ci>RTF</ci>\n"
-        "          <apply>\n"
-        "            <log/>\n"
-        "            <apply>\n"
-        "              <divide/>\n"
-        "              <ci>Nao</ci>\n"
-        "              <ci>Nai</ci>\n"
-        "            </apply>\n"
-        "          </apply>\n"
-        "        </apply>\n"
-        "      </apply>\n"
-        "      <apply>\n"
-        "        <eq/>\n"
-        "        <ci>Na_conductance</ci>\n"
-        "        <apply>\n"
-        "          <times/>\n"
-        "          <ci>g_Na</ci>\n"
-        "          <apply>\n"
-        "            <power/>\n"
-        "            <ci>m</ci>\n"
-        "            <apply>\n"
-        "              <times/>\n"
-        "              <cn cellml:units=\"dimensionless\">3</cn>\n"
-        "              <ci>h</ci>\n"
-        "            </apply>\n"
-        "          </apply>\n"
-        "        </apply>\n"
-        "      </apply>\n"
-        "      <apply>\n"
-        "        <eq/>\n"
-        "        <ci>i_Na</ci>\n"
-        "        <apply>\n"
-        "          <times/>\n"
-        "          <ci>Na_conductance</ci>\n"
-        "          <apply>\n"
-        "            <minus/>\n"
-        "            <ci>V</ci>\n"
-        "            <ci>E_Na</ci>\n"
-        "          </apply>\n"
-        "        </apply>\n"
-        "      </apply>\n"
-        "    </math>\n"
-        "  </component>\n"
+        "  <component name=\"imported_heirarchy\"/>\n"
```

Comment:
Spelling mistake (and the same spelling mistake in other places in this PR).

Reply:
Done